templates/api/knowledge-panels/health/ingredients/ingredients_rare_crops.tt.json

```diff
@@ -0,0 +1,29 @@
+{
+    "level": "info",
+    "topics": [
+        "health"
+    ],
+    "expanded": false,
+    "evaluation": "good",
+    "title_element": {
+        "title":"[% lang("knowledge_panels_ingredients_rare_crops_title") %]",
+        "subtitle": "[% lang("knowledge_panels_ingredients_rare_crops_subtitle") %] [% display_taxonomy_tags_list("ingredients", panel.ingredients_rare_crops) %]",
+        "icon_url": "[% static_subdomain %]/images/panels/divinfood/divinfood.png",
+    },
+    "elements": [
+        {
+            "element_type": "text",
+            "text_element": {
+                "html":
+                `
+<b>[% lang("knowledge_panels_did_you_know") %]</b>
+<p>[% lang("knowledge_panels_ingredients_rare_crops_did_you_know") %]</p>
+<b>[% lang("knowledge_panels_what_you_can_do") %]</b>
```

Comment:
I would add a section about risks of low crop biodiversity.

Crop diversity: an insurance policy for food security. Just like a diverse investment portfolio protects your savings, a diverse range of crops protects our food supply from pests, diseases, and climate change. Monocultures are vulnerable, diversity is resilience.

Reply:
I'll add a section about why it matters. I'm not too sure about the savings metaphor.

Reply:
me neither, a LLM artefact :-)